src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -387,6 +399,36 @@ public function preprocessArticles($linkedArticles, $settings)
         return $articles;
     }
 
+    /**
+     * Preprocess the editorial label for search results
+     * @param string $cdbid
+     * @param string $settings
+     */
+    public function preprocessEditorialLabel($cdbid, $settings)
+    {
+        $articles = $this->curatorenClient->searchArticles($cdbid);
+
+        if (empty($articles['hydra:member'])) {
+            return;
```

Comment:
The return type hint of this method claims it always returns a string, yet we return an implicit `null` here.
Either:
- Change the return type to `string|null` and make the `return null` explicit here
- Return an empty string `return '';`

Reply:
@jdrieghe jup thx for the remark, fixed it.

Reply:
Don't forget to replace the `return;`'s in this method with `return null;` now